site/oauth2-examples-proxy.md

```diff
@@ -0,0 +1,75 @@
+# Use OAuth2 Proxy and Keycloak as OAuth 2.0 server
+
+Let's test the following flow:
+
+* Access management ui via a browser thru OAuth2-Proxy
+
+<pre class="lang-plain">
+                    [ Keycloak ] 3. authenticate
+                      /|\  |
+                       |   | 4. token
+        2.redirect     |  \|/                                        [ RabbitMQ ]
+                [ Oauth2-Proxy ]       ----5. forward with token-->  [  http    ]
+                      /|\
+                       |
+            1. rabbit_admin from a browser
+</pre>
+
+## Prerequisites to follow this guide
+
+- Docker
+- make
+
+## Deploy Key Cloak
+
+First, deploy **Key Cloak**. It comes preconfigured with all the required scopes, users and clients.
```

Comment:
Change to "Deploy Key Cloak (I don't think Key Cloak needs to be bolded like it is above) by running the following command: 

Then after the command, you can say add a note to say: 

Note: Key Cloak is preconfigured with the required scopes, users, and clients. It is configured with its own signing key and the [rabbitmq.conf](https://github.com/rabbitmq/rabbitmq-oauth2-tutorial/tree/main/conf/oauth2-proxy/rabbitmq.conf) file is also configured with the same signing key.